pkg/api/v1beta3/dynakube/dynakube_types.go

```diff
@@ -76,6 +76,9 @@ type DynaKubeSpec struct { //nolint:revive
 	// +kubebuilder:validation:Optional
 	Kspm *kspm.Spec `json:"kspm,omitempty"`
 
+	// +kubebuilder:validation:Optional
+	Extensions *ExtensionsSpec `json:"extensions,omitempty"`
```

Comment:
why was this reordered?

Reply:
Because of the field alignment in the struct. Seems like all the pointer members have to cuddle on the top.